docs/words.txt

```diff
@@ -23,6 +23,8 @@ Redis
 serializable
 serialized
 serializer
+subreddit
```

Comment:
nono, this is totally right and fine. I was just laughing that we didn't have this already.